app/code/Magento/Catalog/Controller/Adminhtml/Product/Action/Attribute/Save.php

```diff
@@ -15,6 +15,19 @@
 class Save extends \Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute
 {
     /**
+     * @var \Magento\Framework\Stdlib\DateTime\TimezoneInterface
+     */
+    protected $localeDate;
+    
+    /**
+     * Eav config
+     *
+     * @var \Magento\Eav\Model\Config
+     */
+    protected $_eavConfig;
```

Comment:
Avoid creation of new properties with leading underscore. Here it also looks like you use it without underscore.